docker-compose.yml

```diff
@@ -14,11 +14,12 @@ services:
       - dbdata:/var/lib/postgresql/data
 
   rabbitmq:
-    image: rabbitmq
+    image: rabbitmq:3-management
```

Comment:
Comment that we need the management version to monitor the gpu task queue.  Can we just specify `rabbitmq:management`?